data/controller_data.py

```diff
@@ -84,6 +84,12 @@
      'image_url': 'http://zero88.com/images/product-pics-scd-server.jpg',
      'tags': ['console']},
   ],
+  0x2239: [
+    {'name': 'LightFactory',
+     'link': 'https://www.lightfactory.net/',
+     'image_url': 'https://www.lightfactory.net/images/lightfactory_square1_small.png'},
```

Comment:
You've left the old curly bracket at the end of this line:
https://travis-ci.org/OpenLightingProject/rdm-app/jobs/507611257

Reply:
Fixed.